ng-ncc-app/src/app/services/uhtrigger.service.ts

```diff
@@ -55,18 +55,14 @@ export class UHTriggerService {
     }
 
     /**
-     * Called when a comms template has been sent to the caller.
+     * Called when a payment has been made against a caller's account.
      */
-    madePayment(payment_reference: string, amount: number) {
-        console.log(payment_reference, amount);
-
+    madePayment(amount: number, payment_reference: string) {
         // Only continue if there is an identified caller.
-        if (!this.Call.isCallerIdentified()) {
-            return;
+        if (this.Call.isCallerIdentified()) {
+            console.log('Recording payment:', payment_reference, amount);
```

Comment:
Are we leaving console log messages in the app?

Reply:
There shouldn't appear in the "live-test" or live versions, for sure. They should be removed, however I'll consider adding this fix: https://stackoverflow.com/questions/42307317/stripping-all-comments-and-console-logs-with-ng-build-prod-possible

Reply:
Good idea! Let's try it and see if it works for live then.